build.gradle

```diff
@@ -833,6 +837,32 @@ project('spring-xd-module-spi') {
 	}
 }
 
+project('spring-xd-spark-streaming') {
+	description = 'Spring XD spark streaming module'
+	dependencies {
+		compile("org.apache.spark:spark-streaming_$sparkScalaVersion:$sparkVersion") {
+			exclude group: 'hsqldb'
+		}
+		compile "org.springframework.integration:spring-integration-core"
+		compile project(":spring-xd-module-spi")
+	}
+}
+
+project('spring-xd-spark-streaming-tests') {
```

Comment:
This was to avoid a dependency cycle, but related to your comment immediately above, it would be ideal to avoid dependencies on the spark code from dirt. However, that will require significant changes to ModuleFactory and others that are beyond the scope of this PR (and unfortunately most likely beyond the scope of 1.1).